beeline/middleware/flask/__init__.py

```diff
@@ -29,7 +29,8 @@ def _teardown_request(self, exception):
             beeline.add_field('request.error_detail',
                               beeline.internal.stringify_exception(exception))
             beeline.add_field('request.error', str(type(exception)))
-            beeline.internal.send_event()
+
+        beeline.internal.send_event()
```

Comment:
I'm not 100% sure of the internals of the beeline and how it works but does this mean we would end up either closing the trace or sending the event twice? If so would this be a problem?

My thinking behind it happening twice is from looking at this block https://github.com/honeycombio/beeline-python/blob/7c5268e2855759f39d443ffef3291c37714c4178/beeline/middleware/flask/__init__.py#L46-L52
If the request is not a 500 then we manually end the [trc](https://github.com/honeycombio/beeline-python/blob/7c5268e2855759f39d443ffef3291c37714c4178/beeline/middleware/flask/__init__.py#L49-L50)
Then after this, if signals support is enabled this `_teardown_request` method would also fire and `beeline.internal.send_event()` would trigger?

Reply:
Hm, thanks for the feedback! We'll take another look at this.